web3/net.py

```diff
@@ -1,41 +1,44 @@
 from typing import (
     NoReturn,
 )
-import warnings
 
-from web3._utils.rpc_abi import (
-    RPC,
+from web3._utils.net import (
+    listening,
+    peer_count,
+    peerCount,
+    version,
 )
 from web3.module import (
-    Module,
+    ModuleV2,
 )
 
 
-class Net(Module):
+class Net(ModuleV2):
     """
         https://github.com/ethereum/wiki/wiki/JSON-RPC
     """
 
+    __listening = listening
```

Comment:
We typically use the pattern of just one underscore for these.

Reply:
sure no problem